README.md

```diff
@@ -26,6 +29,30 @@ In its current state, the swarm will:
 - Be received by the tickRecorder service
 - Write the data point into InfluxDB
 
+## Things that you can do with this setup
+### Micro Web Console
+
+http://localhost:8082
```

Comment:
Every point where you have `localhost`, it'll need to be the $DOCKER_IP. OS X and non-linux machines won't be able to access the docker machines on localhost. Will need instructions on how to get this at the beginning. 

This is also assuming nats container and web and influx containers all share the same docker machine.

===

## Things that you can do with this setup
First, find your Docker IP.
```echo  $(docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' niifinance_nats_1)```

===